### PR TITLE
map borough to areas

### PR DESCRIPTION
**Pull request by PythonGurl**

### CODE REVIEW

File: mapping.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import pandas as pd
+import collections
+
+df = pd.read_csv('RentData051022/medianAskingRent_All.csv')
+print(df.shape)
```
Comment (nerner-code) [inline mapping.py:5]: Lets delete this line

File: mapping.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+def borough_area(df, key, val):
+    d = collections.defaultdict(list)
+    for row in range(df.shape[0]):
+        k = df.at[row, key]
+        v = df.at[row, val]
+        d[k].append(v)
+    return d
+
+
+d = borough_area(df, 'Borough', 'areaName')
```
Comment (nerner-code) [inline mapping.py:17]: Lets get rid of these lines too.  If you want to have some code that will let you test this function, consider putting it in a block like this:

```
if __name__ == "__main__":
    d = borough_area(df, 'Borough', 'areaName')
    print(d)
```

**Review by nerner-code — COMMENTED** [review]
Can you screenshot the output of `borough_area`?

**Comment by PythonGurl** [comment]
image

File: mapping.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import pandas as pd
+import collections
+
+df = pd.read_csv('RentData051022/medianAskingRent_All.csv')
```
Comment (nerner-code) [inline mapping.py:4]: Let's also move this line into the function.  Also, lets drop any rows where `Borough` or `areaName` is null.

**Comment by nerner-code** [comment]
I added some inline comments for your review.

File: mapping.py (hunk 1 above)
Comment (PythonGurl) [reply mapping.py:5]: why?

File: mapping.py (hunk 3 above)
Comment (PythonGurl) [reply mapping.py:4]: why?

File: mapping.py (hunk 2 above)
Comment (PythonGurl) [reply mapping.py:17]: we don't need a return?

**Comment by PythonGurl** [comment]
`defaultdict(<class 'list'>, {'Manhattan': ['All Downtown', 'All Midtown', 'All Upper East Side', 'All Upper Manhattan', 'All Upper West Side', 'Battery Park City', 'Central Harlem', 'Central Park South', 'Chelsea', 'Chinatown', 'East Harlem', 'East Village', 'Financial District', 'Flatiron', 'Gramercy Park', 'Greenwich Village', 'Hamilton Heights', 'Inwood', 'Little Italy', 'Lower East Side', 'Manhattan', 'Midtown', 'Midtown East', 'Midtown South', 'Midtown West', 'Morningside Heights', 'Nolita', 'Roosevelt Island', 'Soho', 'Tribeca', 'Upper East Side', 'Upper West Side', 'Washington Heights', 'West Village'], 'Queens': ['Astoria', 'Central Queens', 'Forest Hills', 'Long Island City', 'Northeast Queens', 'Northwest Queens', 'Queens', 'Rego Park', 'Sunnyside'], 'Brooklyn': ['Bay Ridge', 'Bedford-Stuyvesant', 'Boerum Hill', 'Brooklyn', 'Brooklyn Heights', 'Bushwick', 'Carroll Gardens', 'Clinton Hill', 'Cobble Hill', 'Crown Heights', 'DUMBO', 'Downtown Brooklyn', 'East Brooklyn', 'Fort Greene', 'Greenpoint', 'Greenwood', 'North Brooklyn', 'Northwest Brooklyn', 'Park Slope', 'Prospect Heights', 'Prospect Park', 'South Brooklyn', 'Williamsburg', 'Windsor Terrace'], 'Bronx': ['Bronx', 'Riverdale']})
(env) alyssa@Alyssas-Air Dashboard % git status`

File: mapping.py (hunk 3 above)
Comment (nerner-code) [reply mapping.py]: Import should stay at the top of the file.  We'd just want to move `df = pd.read_csv('RentData051022/medianAskingRent_All.csv')` inside the function.  Its a best practice to not load dataframes in the main scope of a file.

File: mapping.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import collections
+from operator import index
```
Comment (nerner-code) [inline mapping.py:2]: Are you using this (`from operator import index`) anywhere?  If not, we should delete it.

Comment (PythonGurl) [reply mapping.py]: removed

**Review by nerner-code — APPROVED** [review]

**Pull request merged**